oxide-auth/src/code_grant/accesstoken.rs

```diff
@@ -199,7 +199,7 @@ pub enum Input<'req> {
     /// Positively answer an authentication query.
     Authenticated,
     /// Provide the queried refresh token.
-    Recovered(Option<Grant>),
+    Recovered(Box<Option<Grant>>),
```

Comment:
This should probably be an `Option<Box<Grant>>` which has the same size but a much cheaper `None` representation.

Reply:
Done, I've boxed all the Grant for uniformity and push the deref as late as possible.